ComposeStarter/app/build.gradle.kts

```diff
@@ -0,0 +1,127 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+plugins {
+    id ("com.android.application")
+    id ("org.jetbrains.kotlin.android")
+    alias (libs.plugins.roborazzi)
+}
+
+android {
+    compileSdk = 34
+
+    namespace = "com.example.android.wearable.composestarter"
+
+    defaultConfig {
+        applicationId = "com.example.android.wearable.composestarter"
+        minSdk = 26
+        targetSdk = 33
+        versionCode = 1
+        versionName = "1.0"
+    }
+
+    buildTypes {
+        release {
+            isMinifyEnabled = true
+            isShrinkResources = true
+            signingConfig = signingConfigs.getByName("debug")
+            proguardFiles(getDefaultProguardFile("proguard-android-optimize.txt"), "proguard-rules.pro")
+        }
+    }
+
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_17
+        targetCompatibility = JavaVersion.VERSION_17
+        isCoreLibraryDesugaringEnabled = true
+    }
+    testOptions {
+        unitTests {
+            isIncludeAndroidResources = true
+        }
+    }
+    kotlinOptions {
+        jvmTarget = JavaVersion.VERSION_17.majorVersion
+    }
+    buildFeatures {
+        compose = true
+    }
+    composeOptions {
+        kotlinCompilerExtensionVersion = libs.versions.compose.compiler.get()
+    }
+    packaging {
+        resources {
+            excludes += "/META-INF/{AL2.0,LGPL2.1}"
+        }
+    }
+}
+
+dependencies {
+
+    val composeBom = platform(libs.androidx.compose.bom)
+
+    // General compose dependencies
+    implementation(composeBom)
+    implementation(libs.androidx.activity.compose)
+    implementation(libs.androidx.splashscreen)
+
+    // Compose for Wear OS Dependencies
+    // NOTE: DO NOT INCLUDE a dependency on androidx.compose.material:material.
+    // androidx.wear.compose:compose-material is designed as a replacement not an addition to
+    // androidx.compose.material:material. If there are features from that you feel are missing from
+    // androidx.wear.compose:compose-material please raise a bug to let us know:
+    // https://issuetracker.google.com/issues/new?component=1077552&template=1598429&pli=1
+    implementation(libs.wear.compose.material)
+
+    // Foundation is additive, so you can use the mobile version in your Wear OS app.
+    implementation(libs.wear.compose.foundation)
+    implementation(libs.androidx.material.icons.core)
+
+    // Horologist for correct Compose layout
+    implementation(libs.horologist.composables)
+    implementation(libs.horologist.compose.layout)
+    implementation(libs.horologist.compose.material)
+
+    // Preview Tooling
+    implementation(libs.compose.ui.tooling.preview)
```

Comment:
For follow up - these are confusing because name is truncated and missing "wear".

```
androidx-compose-ui-tooling = { module = "androidx.wear.compose:compose-ui-tooling", version.ref = "compose-ui-tooling" }
```

Reply:
Happy to address in another PR and apply to all samples, what about something like `androidx-wear-compose-ui-tooling`?